samples/subsys/smf/hsm_psicc2/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_LOG=y
+CONFIG_SHELL=y
+
+# Needed for boards that enable RTT backends for logging
+# e.g. nrf52840dk/nrf52840 and any others that enable it
+CONFIG_LOG_BACKEND_RTT=n
```

Comment:
this is really suspicious IMO (albeit likely harmful), I don't see any other sample doing anything similar?

Reply:
Yeah, I'm not keen, but there was a discussion on https://github.com/zephyrproject-rtos/zephyr/pull/70641 and it appears to be the only config item that lets logs appear on boards with `CONFIG_LOG_BACKEND_RTT` enabled that doesn't mess with boards that don't have it enabled.

Reply:
I think it's because I'm using logging and most other samples use `printk()`

Reply:
@anangl maybe you can comment on this?